grafana/provider.go

```diff
@@ -28,6 +30,14 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("GRAFANA_AUTH", nil),
 				Description: "Credentials for accessing the Grafana API.",
 			},
+			"headers": {
```

Comment:
`headers` is too generic, can you rename it to `http_headers`? It also maps better to the environment variable name.

Reply:
Changed.